halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentEnvironment.java

```diff
@@ -134,4 +135,10 @@ public Boolean getUpdateVersions() {
     String upstreamUser = "spinnaker";
     String originUser;
   }
+
+  @Data
+  public static class LivenessProbeConfig {
+    boolean enabled;
```

Comment:
I thought the same thing but left it as lowercase `boolean` because the classes above (`Consul`, `Vault`) use lowercase `boolean enabled` -- I think the `isSet` check in `EditDeploymentEnvironmentCommand` is preventing it from being reset to null if only the delay seconds are passed in. I just tested to make sure it works, so I think we're okay :woman_shrugging:

Reply:
Sounds good, I stand corrected!